recipes/mlst/meta.yaml

```diff
@@ -10,10 +10,17 @@ source:
   sha256: '{{ sha256 }}'
 
 build:
-  number: 0
+  number: 1
 
   noarch: generic
 requirements:
+  host:
```

Comment:
any reason why this is now needed?

Reply:
ping

Reply:
@bgruening sorry to be dense, but I am not sure I understand the question. I have tested locally and it appears that the host is required with the python deps. And if I leave the noarch: generic out the build does not pass linting. Can you please provide some info re exactly what you are referring to?

Reply:
@bgruening I was following the skeleton for recipes.